guides/v2.0/contributor-guide/contributing_docs_suggested.md

```diff
@@ -7,7 +7,14 @@ github_link: contributor-guide/contributing_docs_suggested.md
 
 Our goal is to provide the Magento community with comprehensive and quality technical documentation. We believe that to accomplish that goal we need developer experts from the community to contribute to the documentation.
 
-If you are interested in working on one of these projects, open a [pull request on this page](https://github.com/magento/devdocs/tree/develop/guides/v2.0/contributor-guide/contributing_docs_suggested.md){:target="\_blank"}. You can also email us at <DL-Magento-Doc-Feedback@magento.com> or post a comment in the GitHub issue linked to the project title.
+If you are interested in working on one of these projects, open a [pull request on this page](https://github.com/magento/devdocs/tree/develop/guides/v2.0/contributor-guide/contributing_docs_suggested.md){:target="_blank"}. You can also email us at <DL-Magento-Doc-Feedback@magento.com>, chat us at Community Engineering [#devdocs](https://magentocommeng.slack.com/messages/CAN932A3H){:target="_blank"} Slack channel, or post a comment in the GitHub issue linked to the project title.
```

Comment:
"chat with us in the Community Engineering Slack channel"

Reply:
edited